etc/kayobe/inventory/group_vars/github-runners/runners.yml

```
@@ -0,0 +1,38 @@
+---
+runner_user: stack
+github_account: stackhpc
+github_repo: stackhpc-kayobe-config
+access_token: "{{ secrets_github_access_token }}"
+
+base_runner_dir: /opt/actions-runner
+
+default_runner_labels:
+  - kayobe
+  - openstack
+  - "{{ kayobe_environment | default(omit) }}"
```

This variable has a default of an empty string in Kayobe, so shouldn't be undefined.
```suggestion
  - "{{ kayobe_environment }}"
```